src/BmsData.cpp

```diff
@@ -338,13 +338,15 @@ void setBmsChargePercentage(uint8_t devNr, uint8_t value)
       bo_SOC100CellvolHasBeenReached[devNr]=true;
       value=100;
     }
-    else if(u16_CellvoltSoc100>0 && u16_CellvoltSoc0>0){
+    else if((u16_CellvoltSoc0 > 0) &&
```

Comment:
@shining-man Can you please confirm that this change does not invent another issue.
The problem here is probably more of a theoretical nature, but can occur if both values are set to the same value.
SOC100 should always be greater than SOC0 anyway, so in my opinion it should fit.

Reply:
That's ok. That shouldn't be a problem.